test/tests/operations/PGP.js

```diff
@@ -0,0 +1,457 @@
+/**
+ * PGP tests.
+ *
+ * @author tlwr [toby@toby.codes]
+ *
+ * @copyright Crown Copyright 2017
+ * @license Apache-2.0
+ */
+import TestRegister from "../../TestRegister.js";
+
+var CYBERCHEF_GENERATED_KEY_PAIRS = [
+    {
+        name: "CyberChef 1",
```

Comment:
You can put all these in a  .json file and import it. 

Also it's that var again, use const